mne/io/base.py

```diff
@@ -1500,6 +1500,31 @@ def plot_psd_topo(self, tmin=0., tmax=None, fmin=0, fmax=100, proj=False,
                                  axis_facecolor=axis_facecolor, dB=dB,
                                  show=show, n_jobs=n_jobs, verbose=verbose)
 
+    def plot_sensors(self, kind='topomap', show=True):
```

Comment:
see how evoked.plot_topomap does it. we generally display meg by default if present otherwise eeg.